modules/virt-querying-metrics.adoc

```diff
@@ -61,6 +61,7 @@ topk(3, sum by (name, namespace) (rate(kubevirt_vmi_network_receive_bytes_total[
 [id="virt-promql-storage-metrics_{context}"]
 == Storage metrics
 
+[discrete]
```

Comment:
IIRC, we aren't supposed to use `[discrete]` moving forward (for DITA reasons). I think we should change sections like this (with === or ==== subsections) into description lists. They can be fairly complex (see https://docs.asciidoctor.org/asciidoc/latest/lists/description/). @jhradilek - is my understanding correct?

Reply:
@ousleyp You are absolutely right, both the discrete heading and the [level 2 section headings](https://docs.asciidoctor.org/asciidoc/latest/sections/titles-and-levels/) are problematic and your proposed solution is the correct one.

Reply:
@Vaishali-gif-rh Can you please look into the nested description list syntax? The content is not rendering correctly. You might need to wrap lines 64-76 under "Storage-related traffic" in an [open block](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#enclose-in-open-block) (`--`).